src/main/kotlin/racingcar/RacingCarApplication.kt

```diff
@@ -2,21 +2,14 @@ package racingcar
 
 import racingcar.domain.Car
 import racingcar.domain.Cars
-import racingcar.view.InputView
-import racingcar.view.ResultView
+import racingcar.domain.RacingGame
+import racingcar.view.*
 
 fun main() {
-    val inputView = InputView()
-    val resultView = ResultView()
-    val cars = Cars()
-    val carNames = inputView.getCarNames()
-    cars.carList = carNames!!.map { name -> Car(name) }
-    var trial = inputView.getHowManyTrial()
-    resultView.showResult()
-    while (trial!! > 0) {
-        cars.moveCars()
-        resultView.showEachResults(cars)
-        trial--
-    }
-    resultView.showFinalResult(cars.getWinnerList())
+    val cars = Cars(getCarNames().map { name -> Car(name) }.toList())
```

Comment:
```suggestion
    val carNames = getCarNames().map { name -> Car(name) }
            .toList()
    val cars = Cars(carNames)
```

이런 식으로 변경하면 어떨까?
Cars 생성자가 어떠한 인자를 받는 지 좀 더 명확히 드러낼 수 있을 것 같아.

그리고 추가적으로 저렇게 Chaining Method의 경우 `.` 를 기준으로 개행을 하면 더 가독성을 높일 수도 있어 👍

Reply:
요거 또한 밑에서(맞나..?) 언급한 부분처럼 변수로 나누는게 더 가독성있고 의미 있어 보이겠네 ㅎㅎ